index.html

```diff
@@ -0,0 +1,172 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <link rel="stylesheet" type="text/css" href="styles/styles.css">
+  <title>Adoptadog</title>
+</head>
+
+<body>
+
+  <header>
+
+    <img src="https://raw.githubusercontent.com/Sophiagts/html200-adopt-a-dog/41dab5423bd1111155d9d5bd7f6a44c57e8119e2/images/adoptadog-logo.svg" alt="adoptadog logo">
+
+    <h1>Seattle’s premiere dog adoption agency</h1>
+
+
+    <img src="https://raw.githubusercontent.com/Sophiagts/html200-adopt-a-dog/41dab5423bd1111155d9d5bd7f6a44c57e8119e2/images/cart-icon.svg" alt="shopping cart icon">
+
+    <p>$1234.56</p>
+
+    <a href="./contact.html">
+      <button type="button" name="Checkout">Checkout</button>
+    </a>
+
+    <nav>
+      <ul>
+        <li><a href="./index.html">Home</a></li>
+        <li><a href="./dogs.html">Dogs</a></li>
+        <li><a href="./blog.html">Blog</a></li>
+      </ul>
+    </nav>
+  </header>
+
+  <section class="container">
+
+
+
+    <article class="">
```

Comment:
You'll have to figure out a way to lay this element out of the flow of the dog tiles